conf/services/apply-for-action-to-protect-your-intellectual-property-rights.yml

```diff
@@ -0,0 +1,34 @@
+serviceName: apply-for-action-to-protect-your-intellectual-property-rights
+serviceHeaderName: apply-for-action-to-protect-your-intellectual-property-rights
+serviceDescription: |
```

Comment:
Could you please check **serviceName** and **serviceHeaderName** ? It should be a statement like below yml files.